less/new-flow-test/thank-you.less

```diff
@@ -1,90 +0,0 @@
-.base-line-thank-you,
-.base-line-share {
-  display: none;
-}
-
-.new-flow-thank-you,
-.new-flow-share {
-  display: inline;
-  .container,
-  .header {
-    text-align: center;
-  }
-  .header {
-    margin-bottom: 0;
-    margin-top: 22px;
-  }
-  .container {
-    padding-top: 30px;
-    padding-bottom: 40px;
-    h2 {
-      margin-bottom: 17px;
-    }
-  }
-  #facebook,
-  #twitter {
-    margin-top: 15px;
-    width: 162px;
-    height: 149px;
-    cursor: pointer;
-    position: relative;
-    padding: 0;
-    .social-button,
-    a {
-      padding: 25px;
-      position: absolute;
-      width: 100%;
-      height: 100%;
-      font-size: 18px;
-      div {
-        position: absolute;
-        width: 100%;
-        left: 0px;
-        bottom: 22px;
-        margin: 0;
-        width: 100%;
-        font-size: 17px;
-      }
-    }
-  }
-  #facebook {
-    margin-right: 8px;
-    margin-left: 5px;
-    background-color: #406DB4;
-    &:hover {
-      background-color: #4E7AC0;
-    }
-  }
-  #twitter {
-    margin-right: 5px;
-    margin-left: 8px;
-    background-color: #46C6F1;
-    a {
-      i {
-        margin-top: -10px;
-      }
-    }
-    &:hover {
-      background-color: #5ECDF3;
-    }
-  }
-  h1,
-  h2 {
-    line-height: 1.2;
-    letter-spacing: -1px;
-  }
-  h1 {
-    margin: 35px 0;
-    font-size: 32px;
-  }
-  h2 {
-    font-size: 25px;
-  }
-  .heart-image,
-  .internet-graphic {
-    max-width: none;
-  }
-  .submit-btn {
-    margin-top: 25px;
-  }
-}
```

Comment:
This ends the removal of pattern 3. All these styles have been merged into existing css files, or removed if it already existed in an existing css prop.